hyper/compat.py

```diff
@@ -31,6 +38,9 @@ def ignore_missing():
 if is_py2:
     if is_py2_7_9_or_later:
         import ssl
+    elif is_appengine:
+        from . import ssl_compat_appengine
+        ssl = ssl_compat_appengine
```

Comment:
Fixed, use ssl_compat_appengine when appengine.

When an AppEngine runtime got updated to Python 2.7.9 or later, I was personally thinking that it will use the bundled ssl and this compatibility module becomes unnecessary.
Indeed, however, nothing is sure for now, and anyways this would work with bundled ssl as I commented on another place, using it for GAE environment would be at least harmless even in that.